src/preact/storybook/Hooks.js

```diff
@@ -0,0 +1,68 @@
+import {withKnobs} from '@storybook/addon-knobs';
+
+import * as Preact from '#preact';
+import {useCallback, useState} from '#preact';
+import {useIntersectionObserver} from '#preact/component';
+import {refs} from '#preact/utils';
+
+export default {
+  title: '0/Hooks',
+  decorators: [withKnobs],
+};
+
+function Component({prop}) {
+  const [text, setText] = useState('initial render');
+  const ioCallback = useCallback(
+    ({isIntersecting}) => {
+      setText(`is intersecting for ${prop}: ${isIntersecting}`);
+    },
+    [prop]
+  );
+
+  const anotherIoCallback = useCallback(
+    ({isIntersecting}) => {
+      // eslint-disable-next-line local/no-forbidden-terms
+      console.log(`is intersecting for ${prop}: ${isIntersecting}`);
```

Comment:
Are we intentionally having a `console.log` call in a storybook?

Reply:
We're already displaying some text - couldn't think of anything else 😞

Reply:
I think storybooks have something called "Actions" where you could make it call an `onIntersection`?
idk if we use it

https://storybook.js.org/docs/rax/essentials/actions

Reply:
Ah yes, Alan is has started using Actions in some Storybooks. I'm adding a todo for this.